src/main/java/build/buildfarm/rpms/server/BUILD

```diff
@@ -0,0 +1,19 @@
+load("@rules_pkg//:rpm.bzl", "pkg_rpm")
+
+pkg_rpm(
+    name = "buildfarm-server-rpm",
+    data = [
+       "//examples:example_configs",
+       "//src/main/java/build/buildfarm:buildfarm-server_deploy.jar",
+       "//src/main/java/build/buildfarm:configs",
+       ":server-service"
+    ],
+    release = "0",
+    spec_file = "buildfarm-server.spec",
```

Comment:
Instead of writing a spec, [@nacl](https://github.com/nacl) recommended using the newer pkg_rpm.

This will allow the postrun/prerun scriptlets to be just shell scripts that can be included.

He points to the example here: https://github.com/bazelbuild/rules_pkg/blob/main/pkg/tests/rpm/BUILD#L125

Rules provided by https://github.com/bazelbuild/rules_pkg/blob/main/pkg/mappings.bzl

But I guess this improvement can come in as a separate PR.

Reply:
Sure. I will check the newer version of rules_pkg and we can have it in separate PR.